docs/guides/vite.md

```diff
@@ -1093,7 +1088,7 @@ Finally, we were inspired by how other frameworks implemented Vite support:
 [vite-config]: ../file-conventions/vite-config
 [vite-plugins]: https://vitejs.dev/plugins
 [vite-features]: https://vitejs.dev/guide/features
-[supported-remix-config-options]: #configuration
+[supported-remix-config-options]: https://remix.run/docs/en/main/file-conventions/vite-config
```

Comment:
```suggestion
[supported-remix-config-options]: ../file-conventions/vite-config
```
This is the same link as `[vite-config]`, is this intended?

Reply:
Ah, thanks for the catch!